invenio_rdm_records/requests/access/requests.py

```diff
@@ -134,17 +134,21 @@ def execute(self, identity, uow):
         permission = self.request["payload"]["permission"]
 
         data = {
-            "permission": permission,
-            "subject": {
-                "type": "user",
-                "id": str(creator.id),
-            },
-            "origin": f"request:{self.request.id}",
+            "grants": [
+                {
+                    "permission": permission,
+                    "subject": {
+                        "type": "user",
+                        "id": str(creator.id),
+                    },
+                    "origin": f"request:{self.request.id}",
+                }
+            ]
         }
 
         # NOTE: we're using the system identity here to avoid the grant creation
         #       potentially being blocked by the requesting user's profile visibility
-        service.access.create_grant(system_identity, record.pid.pid_value, data)
+        service.access.create_grants(system_identity, record.pid.pid_value, data)
```

Comment:
just cosmetics, please share you opinion before changing: for plural it is easy to make fat finger mistakes therefore maybe it is better we call this method `bulk_create_grants`